mappings/net/minecraft/client/render/entity/LivingEntityRenderer.mapping

```diff
@@ -2,22 +2,45 @@ CLASS net/minecraft/class_922 net/minecraft/client/render/entity/LivingEntityRen
 	FIELD field_21011 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_4737 model Lnet/minecraft/class_583;
 	FIELD field_4738 features Ljava/util/List;
+	METHOD <init> (Lnet/minecraft/class_898;Lnet/minecraft/class_583;F)V
+		ARG 1 dispatcher
+		ARG 2 model
+		ARG 3 shadowSize
 	METHOD method_18656 getYaw (Lnet/minecraft/class_2350;)F
+		ARG 0 direction
 	METHOD method_23185 getWhiteOverlayProgress (Lnet/minecraft/class_1309;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
 	METHOD method_23622 getOverlay (Lnet/minecraft/class_1309;F)I
+		COMMENT Returns the packed overlay color for an entity, determined by its death progress and whether it is flashing.
 		ARG 0 entity
 		ARG 1 whiteOverlayProgress
+	METHOD method_24302 getRenderLayer (Lnet/minecraft/class_1309;ZZ)Lnet/minecraft/class_1921;
+		COMMENT Gets the render layer appropriate for rendering the passed entity. Returns null if the entity should not be rendered.
+		ARG 1 entity
+		ARG 2 showBody
+		ARG 3 translucent
 	METHOD method_4039 getLyingAngle (Lnet/minecraft/class_1309;)F
+		ARG 1 entity
 	METHOD method_4042 scale (Lnet/minecraft/class_1309;Lnet/minecraft/class_4587;F)V
 		ARG 1 entity
+		ARG 2 matrices
+		ARG 3 tickDelta
 	METHOD method_4044 getHandSwingProgress (Lnet/minecraft/class_1309;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
-	METHOD method_4045 getCustomAngle (Lnet/minecraft/class_1309;F)F
+	METHOD method_4045 getAnimationProgress (Lnet/minecraft/class_1309;F)F
+		COMMENT This value is passed to other methods when calculating angles for animation.
+		COMMENT It's typically just the sum of the entity's age (in ticks) and the passed in tickDelta.
```

Comment:
Should we wrap param ref in method doc with `{@code }`, like how `limit` param is wrapped in https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#split-java.lang.String-int-